frameworks/digital-outcomes-and-specialists-5/questions/declaration/readUnderstoodGuidance.yml

```diff
@@ -1,8 +1,14 @@
 name: Understanding the application process
-question: >
-  Do you understand the application process for Digital Outcomes and Specialists&nbsp;5 on the Digital Marketplace?
-hint: >
-  You can read about how to apply in the <a href="https://www.gov.uk/guidance/digital-outcomes-and-specialists-suppliers-guide#how-to-apply" target="_blank" rel="noopener noreferrer">Digital Outcomes and Specialists suppliers’ guide (link opens in a new tab)</a>.
+question: |
+  You can read about how to apply in the <a href="https://www.gov.uk/guidance/digital-outcomes-and-specialists-suppliers-guide#how-to-apply" target="_blank" rel="noopener noreferrer">Digital Outcomes and Specialists suppliers’ guide (opens in new tab)</a>.
+
+  By starting an application for Digital Outcomes and Specialists&nbsp;5 do you understand:
+
+  - the application process
+  - that you must make the supplier declaration and mark at least one service as complete before your application can be considered
+  - that you can ask clarification questions only through the updates page in your Digital Marketplace account
+question_advice: >
+  You can ask clarification questions about Digital Outcomes and Specialists&nbsp;5 until 5pm BST on 30 October 2020. All questions and answers will be published regularly on the updates page. You will be notified by email when new questions and answers are available.
```

Comment:
Putting this content in the question_advice means that it's not immediately clear what I'm answering yes/no to

<img width="711" alt="Screenshot 2020-09-29 at 15 58 40" src="https://user-images.githubusercontent.com/503614/94576190-fbd9a380-026c-11eb-9989-078f9c1693d0.png">

Reply:
Agree.

```suggestion
question_advice: >
  You can ask clarification questions about Digital Outcomes and Specialists&nbsp;5 until 5pm BST on 30 October 2020. All questions and answers will be published regularly on the updates page. You will be notified by email when new questions and answers are available.
```
Move this to below Yes/No buttons.

Reply:
Move opening paragraph to below heading, so the numbering does not apply to it.

Reply:
I can move the opening paragraph up. However, I can't easily move the third paragraph down. I can make it into a hint, though. It now looks like:

![image](https://user-images.githubusercontent.com/15256121/94581403-f717ee00-0272-11eb-83f9-d6495eec9a34.png)

Reply:
LGTM